packages/react-native-aztec/ios/RNTAztecView/RCTAztecView.swift

```diff
@@ -135,6 +135,7 @@ class RCTAztecView: Aztec.TextView {
         textContainerInset = .zero
         contentInset = .zero
         textContainer.lineFragmentPadding = 0
+        frame.size = CGSize(width: 0, height: 0);
```

Comment:
```suggestion
        frame.size = CGSize(width: 0, height: 0)
```

Reply:
```suggestion
        frame.size = .zero
```

Reply:
Just nitpicks 😄

Reply:
Oh good catch! I forgot to run the autocorrect on styling, thanks!